tests/msgpack.light.tests/Generators/ClassInheritanceTest.cs

```diff
@@ -1,22 +1,18 @@
 ﻿using System;
+using System.Collections;
+using System.Collections.Generic;
 
 using Shouldly;
 
 using Xunit;
 
 namespace ProGaudi.MsgPack.Light.Tests.Generators
 {
-    public class ClassInheritanceTest : MapGeneratorTestBase, IClassFixture<ClassInheritanceTest.Fixture>
+    public class ClassInheritanceTest : MapGeneratorTestBase
     {
-        private readonly Fixture _fixture;
-
-        public ClassInheritanceTest(Fixture fixture)
-        {
-            _fixture = fixture;
-        }
-
-        [Fact]
-        public void WriteSmoke()
+        [Theory]
+        [ClassData(typeof(FixtureProvider))]
```

Comment:
The idea was to provide one time initialization for context. Why should we drop it?